src/JuMP.jl

```diff
@@ -32,8 +32,8 @@ export
     affToStr, quadToStr, conToStr, chgConstrRHS,
 
 # Macros and support functions
-    @addConstraint, @addConstraints, @defVar,
-    @defConstrRef, @setObjective, addToExpression, @defExpr,
+    @addConstraint, @defConstraint, @addConstraints, @defConstraints,
```

Comment:
I have some code that treats constraints as first-class (add them to formulations, use in callbacks, etc), so it's nice to have syntax to construct them without adding them to a model.